src/persist-plugins/mmkv.ts

```diff
@@ -85,7 +85,11 @@ export class ObservablePersistMMKV implements ObservablePersistPlugin {
         const v = this.data[table];
         if (v !== undefined) {
             try {
-                storage.set(table, safeStringify(v));
+                if (v !== null) {
+                    storage.delete(table);
+                } else {
+                    storage.set(table, safeStringify(v));
```

Comment:
Note: I find `safeStringify()` a bit strange as it's implemented as this:

```return value ? JSON.stringify(value, replacer) : value;```

It means that if the value is `false`, `0` or `""`, it won't be serialized as a string.
I'm not sure I understand the intent here, I would expect it to always return a string or `undefined`.